recipes/fmpy/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "fmpy" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/CATIA-Systems/FMPy.git
+  git_tag: "v{{ version }}"
+
+build:
+  skip: true  # [not win]
```

Comment:
Why are the linux and osx platforms being skipped? I see a travis ci config in the repo and it says "runs on Windows, Linux and macOS" in the README.

Reply:
Oh sorry, another relict from our company's recipe. We are mainly using Windows and I copied this line from our internal recipe. Thanks for the catch, I'll remove the skip.